test/Microsoft.DocAsCode.Build.Engine.Tests/ValidateBookmarkTest.cs

```diff
@@ -12,21 +12,35 @@ namespace Microsoft.DocAsCode.Build.Engine.Tests
     using Microsoft.DocAsCode.Build.Common;
     using Microsoft.DocAsCode.Common;
     using Microsoft.DocAsCode.Plugins;
+    using Microsoft.DocAsCode.Tests.Common;
 
     using Xunit;
 
-    public class ValidateBookmarkTest
+    [Collection("docfx STA")]
+    public class ValidateBookmarkTest : TestBase
     {
+        private readonly string _outputFolder;
         private LoggerListener _listener = new LoggerListener("validate_bookmark.ValidateBookmark");
 
+        public ValidateBookmarkTest()
+        {
+            _outputFolder = GetRandomFolder();
+            EnvironmentContext.SetBaseDirectory(_outputFolder);
+            EnvironmentContext.SetOutputDirectory(_outputFolder);
+        }
+
+        public override void Dispose()
+        {
+            EnvironmentContext.Clean();
```

Comment:
We cannot move it to TestBase, unless all test base test is marked collection docfx sta.